src/resolvers/ReviewResolver.ts

```diff
@@ -0,0 +1,277 @@
+    }
+
+    // Question Mutations
+    @Mutation(() => ReviewQuestion)
+    @UseMiddleware(isAuth)
+    async addQuestionToCampaign(@Arg("input") input: AddQuestionToCampaignInput) {
+        const campaign = await ReviewCampaign.findOne({ where: { id: input.campaignId } });
+        if (!campaign) {
+            throw new Error("Review campaign not found");
+        }
+
+        const question = ReviewQuestion.create({
+            question: input.question.question,
+            questionType: input.question.questionType,
+            rangeMin: input.question.rangeMin,
+            rangeMax: input.question.rangeMax,
+            reviewCampaign: campaign
+        });
+
+        await question.save();
+
+        // Create dropdown options if needed
+        if (input.question.dropDown && input.question.questionType === "dropdown") {
+            for (const dropDownInput of input.question.dropDown) {
+                const dropDown = DropDown.create({
+                    text: dropDownInput.text,
+                    value: dropDownInput.value,
+                    reviewQuestion: question
+                });
+                await dropDown.save();
+            }
+        }
+
+        return question;
+    }
+
+    @Mutation(() => ReviewQuestion)
+    @UseMiddleware(isAuth)
+    async updateQuestion(@Arg("input") input: UpdateQuestionInput) {
+        const question = await ReviewQuestion.findOne({ where: { id: input.id } });
+        if (!question) {
+            throw new Error("Question not found");
+        }
+
+        if (input.question !== undefined) question.question = input.question;
+        if (input.questionType !== undefined) question.questionType = input.questionType;
+        if (input.rangeMin !== undefined) question.rangeMin = input.rangeMin;
+        if (input.rangeMax !== undefined) question.rangeMax = input.rangeMax;
+
+        await question.save();
+
+        // Update dropdown options if provided
+        if (input.dropDown) {
+            // Remove existing dropdown options
+            await DropDown.delete({ reviewQuestion: { id: question.id } });
+
+            // Create new dropdown options
+            for (const dropDownInput of input.dropDown) {
+                const dropDown = DropDown.create({
+                    text: dropDownInput.text,
+                    value: dropDownInput.value,
+                    reviewQuestion: question
+                });
+                await dropDown.save();
+            }
+        }
+
+        return question;
+    }
+
+    @Mutation(() => Boolean)
+    @UseMiddleware(isAuth)
+    async deleteQuestion(@Arg("id") id: string) {
+        const question = await ReviewQuestion.findOne({ where: { id } });
+        if (!question) {
+            throw new Error("Question not found");
+        }
+
+        await question.remove();
+        return true;
+    }
+
+    // Review Mutations
+    @Mutation(() => Review)
+    async submitReview(@Arg("input") input: SubmitReviewInput) {
```

Comment:
review needs to be associated to either people that bought ticket_id, or people registration list